### PR TITLE
Klein_SD PCB support

### PR DESCRIPTION
**Pull request by eerraa**
Klein_SD PCB for SIRIND MontCervin

## Description
Ergonomics Split PCB powered by RP2040.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/era/sirind/klein_sd/info.json
Comment (fauxpark) [inline keyboards/era/sirind/klein_sd/info.json:1]: Please run `qmk format-json` on this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by eerraa** [comment]
@fauxpark 
Thank you sir, I just did qmk format-json. is this right?

**Review by waffle87 — APPROVED** [review]

File: keyboards/era/sirind/klein_sd/config.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+#pragma once
+
+/* BACKLIGHT PWM */
+#define BACKLIGHT_PWM_DRIVER PWMD7
+#define BACKLIGHT_PWM_CHANNEL RP2040_PWM_CHANNEL_B
+
+/* RGB Matrix */
+#define WS2812_PIO_USE_PIO1 // Force the usage of PIO1 peripheral, by default the WS2812 implementation uses the PIO0 peripheral
```
Comment (zvecr) [inline keyboards/era/sirind/klein_sd/config.h:24]: Not using another PIO peripheral
```suggestion
```

File: keyboards/era/sirind/klein_sd/config.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#define RGB_MATRIX_DEFAULT_VAL 60
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_MATRIX_KEYPRESSES
```
Comment (zvecr) [inline keyboards/era/sirind/klein_sd/config.h:27]: Now inferred from enabled animations
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/era/sirind/klein_sd/rules.mk
[the remark this answers is not in the document]
Comment (eerraa) [reply keyboards/era/sirind/klein_sd/rules.mk:1]: @lesshonor 
Thank you!

**Pull request merged**